modules/EnsEMBL/Web/Component/Gene/ComparaTree.pm

```diff
@@ -181,19 +181,25 @@ sub content {
 
   # store g1 param in a different param as $highlight_gene can be undef if highlighting is disabled
   my $gene_to_highlight = $hub->param('g1');
+  my $current_gene = $hub->param('g'); # stable ID of current gene
```

Comment:
Did you try $self->object->gene->display_xref->db_display_name?

You need to check for the display_xref, something like the code here: https://github.com/Ensembl/ensembl-webcode/blob/master/modules/EnsEMBL/Web/Component/Gene/GeneSummary.pm#L51

Reply:
@ens-ap5 Thanks for the suggestion. using `display_xref->display_id` worked.